src/features/hotspots/details/HotspotDetails.tsx

```diff
@@ -597,16 +597,13 @@ const HotspotDetails = ({
               marginLeft="m"
               height={24}
             >
-              {hotspot?.status !== undefined &&
-                hotspotSyncStatus?.status !== undefined && (
```

Comment:
@tyler-whitman This line `hotspotSyncStatus?.status !== undefined` is causing the "Syncing" pill to hide when a new hotspot is added. I believe you added that extra check. Do you recall why?

Reply:
Hmm I dont remember why, maybe just extra precaution..